### PR TITLE
debug: impl size

### PR DESCRIPTION
**Pull request by overvenus**
This PR impl size.

### CODE REVIEW

File: src/raftstore/store/debug.rs [hunk 1]
```diff
@@ -79,6 +78,35 @@ impl Debugger {
...
+                let mut sizes = vec![];
+                for cf in cfs {
+                    let mut size = 0;
+                    self.engines
+                        .kv_engine
+                        .scan_cf(cf, start_key, end_key, true, &mut |_, v| {
+                            size += v.len();
+                            Ok(true)
+                        })
+                        .unwrap();
```
Comment (BusyJay) [inline src/raftstore/store/debug.rs]: It's dangerous.

File: src/raftstore/store/debug.rs [hunk 2]
```diff
@@ -79,6 +78,35 @@ impl Debugger {
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn size<'a>(&self, region_id: u64, cfs: &[&'a str]) -> Result<Vec<(&'a str, usize)>> {
```
Comment (overvenus) [inline src/raftstore/store/debug.rs]: Or it takes `cfs: &[String]`, service side[1] would be much simpler. 

[1] https://github.com/pingcap/tikv/pull/2329/files#diff-a0345b86052a238929c03760ee459f0aR126

Comment (siddontang) [reply src/raftstore/store/debug.rs]: maybe we can use Into<String> ?

File: src/raftstore/store/debug.rs [hunk 3]
```diff
@@ -79,6 +78,42 @@ impl Debugger {
...
+                let start_key = &keys::data_key(region.get_start_key());
+                let end_key = &keys::data_end_key(region.get_end_key());
+                let mut sizes = vec![];
+                for cf in cfs {
+                    let mut size = 0;
+                    box_try!(self.engines.kv_engine.scan_cf(
+                        cf.as_ref(),
+                        start_key,
+                        end_key,
+                        true,
```
Comment (BusyJay) [inline src/raftstore/store/debug.rs]: Should be false.

**Comment by overvenus** [comment]
PTAL @hicqu

**Comment by hicqu** [comment]
LGTM.

**Pull request merged**